README.md

```diff
@@ -155,6 +155,17 @@ cargo update -p package_name --precise 0.1.1
 
 </details>
 
+## Footprint
```

Comment:
I'm not familiar with `Footprint` as a common term for this (and so don't find it compelling as a section title, as it doesn't strongly indicate what will be in the section).

Maybe "Recommended Cargo Config"?